pytest_reana/plugin.py

```diff
@@ -13,6 +13,7 @@
                        consume_queue, cwl_workflow_with_name,
                        cwl_workflow_without_name, db_engine, default_exchange,
                        default_in_memory_producer, default_queue, default_user,
-                       in_memory_queue_connection, session,
+                       in_memory_queue_connection,
+                       sample_serial_workflow_in_db, serial_workflow, session,
```

Comment:
It seems that `sample_yadage_workflow_in_db` was not exposed. Am I missing something?

Reply:
You are right, looks like I overwrote it when added the serial workflow... I will make a PR and a release commit.

Reply:
Thanks :)